server/db/seed_db.ts

```diff
@@ -35,6 +38,8 @@ import { AdminRepository } from "../src/repositories/admin_repository";
     await patientRepository.addPatient(4);
     await patientRepository.addPatient(5);
 
+    await statusRepository.updatePatientStatusFields(5, { weight: true, temperature: false, otherSymptoms: false });
```

Comment:
Nitpick: The business rule is for these 3 to always be true so might as well make the seed data match

```suggestion
    await statusRepository.updatePatientStatusFields(5, { weight: true, temperature: true, otherSymptoms: true });
```